pkg/ws/report/handlers/vector_update.go

```diff
@@ -29,14 +32,28 @@ func (h *VectorUpdateHandler) Handle(msg common.Message, client interfaces.IRepo
 
 	var vectorUpdateMsg VectorUpdateMessage
```

Comment:
Please move the `VectorUpdateMessage` struct to the dto package:

```suggestion
	var vectorUpdateMsg dto.VectorUpdateMessage
```